x-pack/plugins/alerting/server/alert/create_alert_factory.ts

```diff
@@ -116,6 +123,22 @@ export function createAlertFactory<
           );
         }
       },
+      trimRecovered: (recoveredAlerts: TrimRecoveredOpts[]) => {
```

Comment:
What do you think of moving this to a library function that the task runner calls directly and then using this as a pass through to that library function that is used by the rule registry? That way when we remove the rule registry, we can just remove the pass through.

Reply:
Sure!

Reply:
Resolved in this commit e33b281361df85b651c425f597d80a69193eaa52